cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerHttpHandlerInternal.java

```diff
@@ -87,6 +95,29 @@ public void run(FullHttpRequest request, HttpResponder responder) {
     }
   }
 
+  @GET
+  @Path("/token")
+  public void token(FullHttpRequest request, HttpResponder responder) {
```

Comment:
Do we need FullHttpRequest? or HttpRequest would be sufficient?

Reply:
Done!